package.json

```diff
@@ -51,27 +51,28 @@
     "swig": "~1.4.1"
   },
   "devDependencies": {
-    "supertest": "~0.14.0",
```

Comment:
The alphabetic reordering happened because of `npm install karma-spec-reporter --save-dev`. Sry for the unreadability.